tests/integration/contract-call/locked-wallet.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { RouteUrls } from '@shared/route-urls';
+
+import { SendPage } from '../../page-objects/send-form.page';
+import { WalletPage } from '../../page-objects/wallet.page';
+import { BrowserDriver, createTestSelector, selectTestnet, setupBrowser } from '../utils';
+import { SECRET_KEY_2 } from '@tests/mocks';
+import { SettingsSelectors } from '@tests/integration/settings.selectors';
+import { Page } from 'playwright-core';
+
+jest.setTimeout(60_0000);
+jest.retryTimes(process.env.CI ? 2 : 0);
+
+describe('Locked wallet test', () => {
+  const BEFORE_EACH_TIMEOUT = 600000;
+
+  let browser: BrowserDriver;
+  let walletPage: WalletPage;
+  let mainPage: WalletPage;
+  let sendForm: SendPage;
+  let pages: Page[];
+  let latestPage: Page;
+
+  beforeEach(async () => {
+    browser = await setupBrowser();
+    walletPage = await WalletPage.init(browser, RouteUrls.Onboarding);
+    await walletPage.signIn(SECRET_KEY_2);
+    await walletPage.waitForHomePage();
+    // switch to test net
+    await selectTestnet(walletPage);
+    pages = await WalletPage.getAllPages(browser);
+    mainPage = new WalletPage(pages[1]);
+    await mainPage.clickSignUp();
+
+    await walletPage.page.waitForTimeout(2000);
+    pages = await WalletPage.getAllPages(browser);
+    latestPage = pages[pages.length - 1];
+    sendForm = new SendPage(latestPage);
+    await sendForm.waitForPreview('$account1');
+    await sendForm.clickFirstAccount();
+  }, BEFORE_EACH_TIMEOUT);
+
+  afterEach(async () => {
+    try {
+      await browser.context.close();
+    } catch (error) {}
+  });
+
+  it('when wallet is not locked', async () => {
+    await mainPage.clickContractCall();
+    await walletPage.page.waitForTimeout(2000);
```

Comment:
Is this line needed? Playwright has a lot of features to waiting for elements/events/selectors, to avoid the need to add in some arbitrary wait time periods. So it'd be better to wait for something, rather than just a period of time

Reply:
Yes I believe so, I have been using the suggested timeouts from https://playwright.dev/docs/test-timeouts and stackoverflow. They seem to get the job done for our test and cut down on the flakes and failures. I did try await page.waitForNavigation() but page.waitForTimeout() worked better for me so I used that. Definitely open to suggestions.

Reply:
Just had a read of the docs that describe the `waitForTimeout` method here https://playwright.dev/docs/api/class-frame#frame-wait-for-timeout

> Note that frame.waitForTimeout() should only be used for debugging. Tests using the timer in production are going to be flaky. Use signals such as network events, selectors becoming visible and others instead.

Reply:
Thanks @kyranjamie let me take a look

Reply:
I removed the timeouts as requested